toolkits/hubspot/arcade_hubspot/tools/crm/companies.py

```diff
@@ -0,0 +1,63 @@
+from typing import Annotated, Any, Optional
+
+from arcade.sdk import ToolContext, tool
+from arcade.sdk.auth import OAuth2
+
+from arcade_hubspot.enums import HubspotObject
+from arcade_hubspot.models import HubspotCrmClient
+
+
+@tool(
+    requires_auth=OAuth2(
+        id="arcade-hubspot",
```

Comment:
```suggestion
        id="hubspot",
```

This way it will work with any defined Hubspot auth provider, not just the Arcade Cloud default one.

Reply:
Or better yet, can you add a `HubSpot` class like https://github.com/ArcadeAI/arcade-ai/blob/main/arcade/arcade/core/auth.py#L72-L78